video/video.ino

```diff
@@ -115,10 +115,13 @@ void loop() {
 
 		if (processor->byteAvailable()) {
 			if (drawCursor) {
-				drawCursor = false;
 				do_cursor();
 			}
 			processor->processNext();
+			if (drawCursor || !cursorFlashing) {
+				drawCursor = true;
+				do_cursor();
+			}

```

Comment:
the logic here will now hide away the cursor if it was visible before processing a VDU command.  once the command has been processed, if the cursor was visible, or if we're showing a steady (non-flashing) cursor, then we'll redisplay the cursor

the addition of re-showing the cursor if it had been visible after processing the VDU command ensures that the rhythm of the flashing cursor stays consistent, which was not previously the case.  the previous behaviour meant that the cursor would stay invisible when using the cursor keys, which was kinda naff :grin: